src/Stripe.net/Entities/OAuth/OAuthToken.cs

```diff
@@ -4,31 +4,62 @@ namespace Stripe
 
     public class OAuthToken : StripeEntity<OAuthToken>
     {
-        [JsonProperty("token_type")]
-        public string TokenType { get; set; }
-
-        [JsonProperty("scope")]
-        public string Scope { get; set; }
+        /// <summary>
+        /// <para>
+        /// The access token you can use to make requests on behalf of this Stripe account. Use it
+        /// as you would any Stripe secret API key.
+        /// </para>
+        /// <para>
+        /// This key does not expire, but may be revoked by the user at any time (you'll get a
+        /// <c>account.application.deauthorized</c>
+        /// <see href="https://stripe.com/docs/api/events/types#event_types-account.application.authorized">webhook event</see>
+        /// when this happens).
+        /// </para>
+        /// </summary>
+        [JsonProperty("access_token")]
+        public string AccessToken { get; set; }
 
+        /// <summary>
+        /// <para>
+        /// The live mode indicator for the token. If <c>true</c>, the <see cref="AccessToken"/>
+        /// can be used as a live secret key. If <c>false</c>, the <see cref="AccessToken"/> can be
+        /// used as a test secret key.
+        /// </para>
+        /// <para>Depends on the mode of the secret API key used to make the request.</para>
+        /// </summary>
```

Comment:
Same as a above, we could say that requests in test-only or both live/test would work instead of mentioning the access token.

Reply:
All the XML comments are copied straight from our docs and I'd rather not diverge from them, as I think having conflicting information would be even more confusing.

Reply:
I could say we should edit the API ref or docs then, but fair.